### PR TITLE
[Perldoc] dicomSummary.pl documentation

### PR DESCRIPTION
**Pull request by cmadjar**
### Description

Documentation of `dicom-archive/dicomSummary.pl` using perldoc.

### This PR resolves

- [x] Redmine ticket: https://redmine.cbrain.mcgill.ca/issues/15033

### To review

Read the documentation and make sure it all makes sense.

### CODE REVIEW

File: dicom-archive/dicomSummary.pl [hunk 1]
```diff
@@ -4,6 +4,55 @@
...
+
+-version   : print CVS version number and exit
+
+
+=head1 DESCRIPTION
+
+A tool for producing an informative summary for DICOMs in a given directory
+(scanner information, acquisitions list, acquisitions parameters...). This tool
+can also compare the DICOM data present in two directories or compare the DICOM
+data present in a given directory and what is stored in the database.
```
Comment (nicolasbrossard) [inline dicom-archive/dicomSummary.pl]: ...givern directory **with** what is stored in the database.

File: dicom-archive/dicomSummary.pl [hunk 2]
```diff
@@ -199,11 +248,20 @@
...
-################################################
-Read archive metadata from database
-################################################
-=cut 
+
+=pod
+
+=head3 read_db_metadata($StudyUID)
+
+Reads the information stored in the database for a given StudyUID.
```
Comment (nicolasbrossard) [inline dicom-archive/dicomSummary.pl]: I would say:

Accesses the database and gets the path of the file containing the metadata for the given StudyUID. What do you think?

File: dicom-archive/dicomSummary.pl [hunk 3]
```diff
@@ -199,11 +248,20 @@
...
+
+INPUT: the DICOM Study Instance UID (StudyUID)
+
+RETURNS: path to the file where the result of the query (i.e. the DICOM metadata
```
Comment (nicolasbrossard) [inline dicom-archive/dicomSummary.pl]: Followup to the comment above:

RETURNS: the path of the file containing the metadata for the given StudyUID or `undef` if none is found.

File: dicom-archive/dicomSummary.pl (hunk 1 above)
Comment (cmadjar) [reply dicom-archive/dicomSummary.pl]: done

File: dicom-archive/dicomSummary.pl (hunk 2 above)
Comment (cmadjar) [reply dicom-archive/dicomSummary.pl]: much better :). Thanks!

File: dicom-archive/dicomSummary.pl (hunk 3 above)
Comment (cmadjar) [reply dicom-archive/dicomSummary.pl]: great! Thanks!

**Pull request merged**